### PR TITLE
Prepend package path to AMENT_PREFIX_PATH

### PR DESCRIPTION
**Pull request by Briancbn**
Signed-off-by: Chen Bainian <chenbn@artc.a-star.edu.sg>

### Fixed bug

#59 

### Fix applied

Fixed applied described in https://github.com/ros-infrastructure/rosdep/issues/724#issuecomment-643015997.

Please let me know your thoughts on this, if this is ok, I can submit the fix to the remaining 3 repos.

### CODE REVIEW

File: rmf_traffic/colcon.pkg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": [
+    "share/rmf_traffic/hook/ament_prefix_path.dsv", 
+    "share/rmf_traffic/hook/ros_package_path.dsv",
```
Comment (Briancbn) [inline rmf_traffic/colcon.pkg:4]: I think this is primarily used for ROS1? I just added in regardless. I am not too sure.

**Comment by mxgrey** [comment]
Hi, Bainian, thanks for bringing this to our attention!

It's unfortunate that these extra steps are necessary for plain cmake packages to be recognized by rosdep, but until plain cmake packages have better support, I suppose we'll have to add these items to the build script.

Thanks for the contribution, and it would be great if you have time to add it to the other repos!

**Review by mxgrey — APPROVED** [review]

**Comment by Briancbn** [comment]
@mxgrey  Thanks for review!

> Thanks for the contribution, and it would be great if you have time to add it to the other repos!

I will submit the fix to the other repos.

I am not familiar with the history policy here. Do you want me to do a rebase? Or it is ok as it is?

**Comment by mxgrey** [comment]
No need to do anything for the history. We use squash merging, which takes care of rebasing :+1:

**Pull request merged**